control-services/cli/semaphore.py

```diff
@@ -517,12 +517,12 @@ def setup(self):
                     '"environment_id": ' + str(self.envId) + ', '
                     '"alias": "' + name + '", '
                     '"playbook": "' + self.cfg['content_repo']['playbooks']['destroyClass'] + '", '
-                    '"arguments": "[\\"-e\\", \\"class=#####\\"]", '
+                    '"dynamic_vars": "[{\\"name\\":\\"class\\",\\"required\\":true}]"' + ', '
```

Comment:
I don't mind this here because we can just check that a class exists before removing it. Class size not having a definite structure is something that can cause more errors